Bugsnag.podspec.json

```diff
@@ -30,6 +30,6 @@
   "requires_arc": true,
   "public_header_files": [
     "Source/BSG_KSCrashReportWriter.h",
-    "Source/Bugsnag{,Metadata,Configuration,Breadcrumb,Event,Plugin,Client}.h"
+    "Source/Bugsnag{,Metadata,Configuration,Breadcrumb,Event,App,AppWithState,Device,DeviceWithState,Error,Stackframe,Thread,Plugin,Client}.h"
```

Comment:
I think this should now be split into separate lines (with the path duplication that implies).  Alphabetised as well to help with future diffing.